index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="CSS/style.css" />
+    <title>Natnael Front-End Developer</title>
+</head>
+<body>
+    <div class="first">
+        <header class="header bg-white">
+          <nav class="navbar flex">
+            <div class="navbar__link ">N-A</div>
+            <div class="navbar__menu">
+              <ul class="navbar__menu__ul ul flex">
+                <li class="navbar__menu__li">
+                  <a href="#Portfolio" class="navbar__menu__link">Portfolio</a>
```

Comment:
- [ ] Kindly make sure that your website is responsive on mobile devices
<img width="358" alt="Screen Shot 2022-04-27 at 14 17 22" src="https://user-images.githubusercontent.com/28307960/165516328-c39991c5-fc1c-4619-999b-c8b56f74089f.png">